lib/kubeclient/common.rb

```diff
@@ -51,6 +53,9 @@ def initialize_client(
       validate_auth_options(auth_options)
       handle_uri(uri, path)
 
+      @class_owner = class_owner
+      @entities = nil
```

Comment:
The only goal here is to have all instance variables explicitly mentioned at initialization - it serves no functional purpose. 

cc @abonas